pkg/cvo/sync_worker.go

```diff
@@ -58,6 +58,10 @@ type PayloadRetriever interface {
 	RetrievePayload(ctx context.Context, desired configv1.Update) (PayloadInfo, error)
 }
 
+type ClusterProfileRetriever interface {
```

Comment:
add godoc

Reply:
done :+1:

Reply:
a84ec7a1e1 has a godoc; marking this thread resolved.